doc/Advanced.md

```diff
@@ -644,6 +650,45 @@ The script `o2-qc-repo-move-objects` lets the user move an object, and thus all
+
+QC workflows for asynchronous data reconstructions are listed in [O2DPG/Data/production/qc-workflow.sh](https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/qc-workflow.sh).
+The script includes paths to corresponding QC configuration files for subsystems which take part in the reconstruction.
+All the enabled files are merged into a combined QC workflow.
+Thus, it is crucial that unique keys are used in `tasks`, `checks` and `aggregators` structures, as explained in [Merging multiple configuration files into one](#merging-multiple-configuration-files-into-one).
+Post-processing tasks can be added in the script [O2DPG/DATA/production/o2dpg_qc_postproc_workflow.py](https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/o2dpg_qc_postproc_workflow.py).
+Please see the included example and the in-code documentation for further guidelines in this matter.
+
+Generating and reconstructing simulated data is ran by a framework which organizes specific workflows in a directed acyclic graph and executes them in an order which satisfies all the dependencies and allocated computing resources.
+In contrast to data reconstruction, here, QC workflows are executed separately and pick up corresponding input files.
+For further details, please refer to [Adding QC Tasks to the simulation script](https://github.com/AliceO2Group/O2DPG/tree/master/MC#adding-qc-tasks-to-the-simulation-script).
+
+Data and simulation productions are typically executed on Grid and EPNs, and the outcomes can be inspected in [MonALISA](http://alimonitor.cern.ch/).
+In both cases, QC runs alongside of each subjob and incomplete QC results are stored in files.
+For asynchronous data reconstruction, one file `QC.root` is created.
+Simulation subjobs contain a `QC` directory with separate files for each QC workflow.
+Relevant logs can be found in files like `stdout`, `stderr` as well as archives `debug_log_archive.zip` and `log_archive.zip`.
+
+Once an expected percentage of subjobs completes, several QC merging stages are executed, each producing a merged file for certain range of subjobs.
+The last stage produces the complete file for given masterjob.
+This file is read by the `o2-qc --remote-batch` executable to run Checks on the complete objects and all the results to the QCDB.
+Post-Processing Tasks and associated Checks are executed right after.
+
+Some runs contain too much data to be processed with one masterjob.
+In such case, several masterjobs are run in parallel.
+Each produces a `QC.root` file which contains all the statistics for a masterjob.
+The last masterjob to complete recognizes this fact and merges all `QC.root` into `QC_fullrun.root` and only then uploads the results to QCDB.
+To find it, one can use `alien_find`:
+```
+> alien_find /alice/data/2022/LHC22m/523897/apass1_epn QC_fullrun.root
+/alice/data/2022/LHC22m/523897/apass1_epn/0750/QC/001/QC_fullrun.root
+```
+
+TODO explain how a connection to QCDB is made from Grid sites. 
+
```

Comment:
I assume the instructions about the `aliqa1` machine would be added after it's migrated to a safer machine. or somewhere else? Correct me if I was wrong.

Reply:
we have some internal documentation already, I will send it to you.